pkg/volume/projected/projected.go

```diff
@@ -328,6 +328,13 @@ func (s *projectedVolumeMounter) collectData() (map[string]volumeutil.FileProjec
 			}
 			tp := source.ServiceAccountToken
 
+			// When FsGroup is set, we depend on SetVolumeOwnership to
+			// change from 0400 to 0440.
+			mode := *s.source.DefaultMode
```

Comment:
We explicitly ignore default mode for tokens. Why do we need to change this?

Reply:
the fallback is 0444 when no runAsUser or fsGroup is present.

Reply:
Ugh, I can't read. Thanks.